boot/debug.go

```diff
@@ -85,3 +86,37 @@ func DumpBootVars(w io.Writer, dir string, uc20 bool) error {
 	}
 	return nil
 }
+
+// DebugSetBootVars is a debug helper that takes a list of <var>=<value> entries
+// and sets them for the configured bootloader.
+func DebugSetBootVars(dir string, varEqVal []string) error {
+	opts := &bootloader.Options{
+		NoSlashBoot: dir != "" && dir != "/",
+	}
+	switch dir {
```

Comment:
This logic feels like it should be in a helper `bootloader.OptionsForDir(dir string) *bootloader.Options`. I'm not qualified to review the logic here.